build/_k-restore.shade

```diff
@@ -4,7 +4,7 @@ k-restore
     Restores nuget packages required for k projects. Downloads and executes k sdk tools.
 */}
 
-default restore_options='${E("KPM_restore_options")}'
+default restore_options='${E("NUGET3_restore_options")}'
```

Comment:
Hmm not sure we wanted to do the kpm -> nuget3 rename, did we? Not quite sure where that lands yet... @davidfowl ?